addons/hr_timesheet/models/project.py

```diff
@@ -80,18 +80,26 @@ def _search_is_internal_project(self, operator, value):
             operator_new = 'not inselect'
         return [('id', operator_new, (query, ()))]
 
-    @api.depends('allow_timesheets', 'task_ids.planned_hours', 'task_ids.remaining_hours')
+    @api.depends('allow_timesheets', 'task_ids.planned_hours', 'timesheet_ids')
     def _compute_remaining_hours(self):
-        group_read = self.env['project.task'].read_group(
+        group_read_effective_hours = self.env['account.analytic.line'].read_group(
+            domain=[('project_id', 'in', self.filtered('allow_timesheets').ids),
+                    ('task_id', '!=', False), ('task_id.planned_hours', '!=', False),
```

Comment:
```suggestion
                    ('task_id', '!=', False), ('task_id.planned_hours', '!=', 0.0),
```

Reply:
@IT-Ideas Thanks! In that case, isn't better to do `('task_id.planned_hours', 'not in', [False, 0.0])` since in both case, `has_planned_hours_tasks` will be `False` and  remaining_hours to 0?

Reply:
> @IT-Ideas Thanks! In that case, isn't better to do `('task_id.planned_hours', 'not in', [False, 0.0])` since in both case, `has_planned_hours_tasks` will be `False` and remaining_hours to 0?

Isn't 0.0 the default value for a Float value? <https://github.com/odoo/odoo/blob/87307a9010c6c872fe6c746d5ece378ad90d038f/odoo/fields.py#L1496>

Reply:
mhhh I did not know that `float(False) == 0.0`, so you are right (like often) :slightly_smiling_face: updated :+1: